h2/src/main/org/h2/value/ValueTimestampTimeZone.java

```diff
@@ -201,6 +201,30 @@ public short getTimeZoneOffsetMins() {
         return timeZoneOffsetMins;
     }
 
+    /**
+     * Returns name of the timezone such as UTC or GMT+1:00.
+     *
+     * @return name of the timezone
+     */
+    public String getTimeZoneName() {
```

Comment:
not keen on this because it implies that we are preserving the original timezone name, which we are not.

rather just do:

    TimeZone tz = new TimeZone();
    tz.setRawOffset( valueTZ.getTimeZoneOffsetMins() * 60 * 1000);

Reply:
Fixed. 

TestKillRestartMulti failed on Java 7.